oak-core/src/main/java/org/apache/jackrabbit/oak/query/QueryEngineSettingsService.java

```diff
@@ -77,6 +77,14 @@
                         "the queryPaths of the index is taken into account."
         )
         String getStrictPathRestrictionsForIndexes() default DISABLED_STRICT_PATH_RESTRICTION;
+        
+        @AttributeDefinition(
+                name="Fully qualified class names to ignore when finding caller",
+                description="If non-empty the query engine logs the query statement plus the java package "
+                        + "which executed this query. This java package is the first package in the call trace"
```

Comment:
missing space at the end

```suggestion
                        + "which executed this query. This java package is the first package in the call trace "
```